doRequest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Copyright (c) 2012 Desire2Learn Inc.
+ * Copyright (c) 2019 Desire2Learn Inc.
```

Comment:
Does this actually need to change?

Reply:
lol. dunno, i guess not... i just did because i happened to be there and notice it. I certainly don't care either way

Reply:
What I mean is that I'm pretty sure the copyright term is supposed to start from the first year it's applicable, in which case this wouldn't change.

Reply:
It's hard I guess when it comes to changing works like this, since "year of first publication" is a bit questionable.

Reply:
alrighty, i'll put it back then. 🙈

Reply:
reverted in a62a9d6